manual_flagging/yaml/inputs_initial.yaml

```diff
@@ -2,11 +2,92 @@
 # which will (ideally) be run one time when we first populate the flagging
 # tables in Athena.
 ---
-stat_groups: [
-  "rolling_window",
-  "township_code",
-  "class"
-]
+
+# This value can either be 'res_single_family', 'res_multi_family',
+# 'condos'
+housing_run_type: ["res_single_family", "res_multi_family", "condos"]
+
+stat_groups:
+  current:
+    res_single_family:
+      columns:
+        - "rolling_window"
+        - "geography_split"
+        - "bldg_age_bin"
+        - "char_bldg_sf_bin"
+      sf_bin_specification: [1200, 2400]
+      age_bin_specification: [40]
+      class_filter: [
+        "202",
+        "203",
+        "204",
+        "205",
+        "206",
+        "207",
+        "208",
+        "209",
+        "210",
+        "218",
+        "219",
+        "234",
+        "278",
+        "295"
+      ]
+    res_multi_family:
+      columns:
+        - "rolling_window"
+        - "geography_split"
+        - "bldg_age_bin"
+      age_bin_specification: [20]
+      class_filter: ["211", "212"]
+    condos:
+      columns:
+        - "rolling_window"
+        - "geography_split"
+      class_filter: ["297", "299", "399"]
+  og_mansueto:
+    res_single_family:
+      columns:
+        - "rolling_window"
+        - "township_code"
+        - "class"
+      class_filter: [
```

Comment:
I think I need to bring these `class_filter` lists out and incorporate them with `housing_run_type`